toolkit/tools/pkg/imagecustomizerlib/cosicommon.go

```diff
@@ -47,8 +49,20 @@ func convertToCosi(ic *ImageCustomizerParameters) error {
 		defer os.Remove(path.Join(outputDir, partition.PartitionFilename))
 	}
 
+	buildDir := filepath.Join(outputDir, "build")
+	existingImageConnection, _, _, err := connectToExistingImage(ic.rawImageFile, buildDir, "imageroot", true)
```

Comment:
Calling `connectToExistingImage` here could cause problems, since it might invalidate the verity partitions. It would probably be better to run the query in the `customizeImageHelper` function and then plumb the information here.